Calculator/TestAddition.cs

```diff
@@ -0,0 +1,38 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Calculator
+{
+    [TestFixture]
+    public class TestAddition
+    {
+        private Addition addition;
+        [SetUp]
+        public void SetUp()
+        {
+            addition = new Addition();
+        }
+        [TestCase(1, 2, 3)]
+        [TestCase(0.1,0.2,0.3)]
+        public void Testaddingbothpositive(decimal first, decimal second, decimal result)
```

Comment:
The names of these tests are still inconsistent with the modulus tests. The first letter of each word should be capitalised. For example: TestAddingBothPositive